docs/source/guide/webhook_create.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Create custom events for webhooks in Label Studio
+short: Custom Webhooks
+type: guide
+order: 655
+meta_title: Create Custom Webhooks in Label Studio
+meta_description: Label Studio documentation for creating custom webhook event triggers to create custom integrations between Label Studio and your machine learning pipeline
+---
+
```

Comment:
@chiganov  feel free to add any additional content that you'd like to this doc, and then I can restructure it. otherwise I'll save it for later.

Reply:
I made this pretty for now, so I think it's okay in it's basic form for now.